### PR TITLE
Add Zen Rabanes details

### PR DESCRIPTION
**Pull request by zenrabanes**

### CODE REVIEW

**Comment by Jansalvador1445** [comment]
Has a conflict, please rebase.

**Comment by zenrabanes** [comment]
hellow 👍 😄

**Comment by zenrabanes** [comment]
closing #148

File: assets/data/devs.json [hunk 1]
```diff
@@ -89,28 +90,9 @@
 	"name": "Ma. Grace Q. Carpizo",
 	"roles": ["Flutter Developer PH Community Member"],
 	"about": "Flutter Tinkerer"
-}, {
-	"name": "Zaldy O. Pagaduan Jr.",
-	"roles": ["Flutter Developer PH Community Member"],
-	"about": "Davaoeño"
-}, {
```
Comment (Jansalvador1445) [inline assets/data/devs.json:96]: Do not remove other data, All you need to do it, add yourself to the last part.

**Comment by Jansalvador1445** [comment]
Please rebase, do not remove data, just add. Thanks

File: assets/data/devs.json (hunk 1 above)
Comment (zenrabanes) [reply assets/data/devs.json:96]: Oh my apologies. 😞

Comment (Jansalvador1445) [reply assets/data/devs.json:96]: No Worries! This happens to us also :D

Comment (Jansalvador1445) [reply assets/data/devs.json:96]: Take this as an opportunity to grow! Usually, tech companies do this! So keep it up and cheer up! @zenrabanes

**Comment by Jansalvador1445** [comment]
Same issue, please rebase

**Pull request closed without merging**